pixeltable/exprs/string_expr.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+from typing import Any, Optional, Union
+
+import sqlalchemy as sql
+
+import pixeltable.exceptions as excs
+import pixeltable.type_system as ts
+
+from .data_row import DataRow
+from .expr import Expr
+from .globals import StringOperator
+from .row_builder import RowBuilder
+from .sql_element_cache import SqlElementCache
+
+
+class StringOpExpr(Expr):
```

Comment:
StringOp is enough (everything in here is an Expr).

Also, the file name should mirror the class name.